tensorflow/compiler/mlir/tools/kernel_gen/tf_to_kernel.cc

```diff
@@ -109,7 +109,8 @@ Status Run(llvm::StringRef input_file, llvm::StringRef output_file,
            llvm::ArrayRef<int64_t> tile_sizes,
            llvm::ArrayRef<int64_t> unroll_factors, int64_t max_supported_rank,
            bool embed_memref_prints, bool print_ptx, bool print_llvmir,
-           bool enable_ftz, bool cpu_codegen, bool jit_compile) {
+           bool enable_ftz, bool index_64bit, bool cpu_codegen, 
+           bool jit_compile) {
   // Read TF code.
```

Comment:
Can we throw an error if `index_64bit` is requested as this is currently unimplemented?

Reply:
unimplemented error is added